README.rst

```diff
@@ -331,6 +331,105 @@ the token in the ``Authorization`` header:
     # send bearer token
     http GET localhost:8000/api/v1/firmware/build/ "Authorization: Bearer $TOKEN"
 
+List of endpoints
+~~~~~~~~~~~~~~~~~
+
+Since the detailed explanation is contained in the `Live documentation <#live-documentation>`_
+and in the `Browsable web page <#browsable-web-interface>`_ of each point,
+here we'll provide just a list of the available endpoints,
+for further information please open the URL of the endpoint in your browser.
+
+Change User password
+^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: text
+
+    PUT /api/v1/user/users/{id}/changepassword/
```

Comment:
Hi, GET is allowed, because I am displaying the ID & Username of the user's whose password will be changed. If that is not required, should I remove it and allow PUT request only?

Reply:
I think only PUT should be allowed here, it's the only one that it's really needed.